static/js/admin/src/MoveDropLocation.js

```diff
@@ -50,27 +50,40 @@ define('MoveDropLocation', [ 'jquery', 'jquery-ui', 'ConfirmationDialog'],
 				
 				// Confirm the move operation before performing it
 				var representative = ui.draggable.data("resultObject");
-				
-				var repTitle = representative.metadata.title;
-				if (repTitle.length > 50) {
-					repTitle = repTitle.substring(0, 50) + "...";
-				}
-				
-				var destTitle = metadata.title;
-				if (destTitle.length > 50) {
-					destTitle = destTitle.substring(0, 50) + "...";
+				var promptText = '';
+				var moveSingleObj = self.manager.dragTargets.length === 1;
+
+				var destTitle = self._formatTitle(metadata.title);
+				var targetAdminUnit = self._getAdminUnit(metadata.objectPath);
+
+				var repTitle = self._formatTitle(representative.metadata.title);
+				var currentAdminUnit = self._getAdminUnit(representative.metadata.objectPath);
```

Comment:
I think you'll need to check every object being move to see if it is moving to a different admin unit. You can have objects from multiple admin units in the same search result, in which case we'd particularly want to be able to warn users. If objects are coming from multiple admin units, maybe we can just say "from multiple admin units" rather than showing a title